/ktp_task_controller:
  ros__parameters:
    device_id: "KECDSEMITB001"
    map_id: "966"
